src/internal/fileserver/fileserver_test.go

```diff
@@ -26,6 +26,7 @@ import (
 )
 
 func TestDownload(t *testing.T) {
+	t.Log("hi")
```

Comment:
hello

Reply:
LOL.  I noticed this and did a search for other debugging messages I added.  I think I killed them all 😂